apis/v1alpha2/shared_types.go

```diff
@@ -46,8 +46,9 @@ type ParentReference struct {
 	// +optional
 	Kind *Kind `json:"kind,omitempty"`
 
-	// Namespace is the namespace of the referent. When unspecified (or empty
-	// string), this refers to the local namespace of the Route.
+	// Namespace is the namespace of the referent. When unspecified, this refers
+	// to the local namespace of the Route. Refer to the Namespace type for valid
+	// values.
```

Comment:
```suggestion
	// Namespace is the namespace of the referent. When unspecified, this refers
	// to the local namespace of the Route.
```

I think the "Refer to the Namespace type" bit isn't useful because the docs will include a link to that type. If it was useful, we'd have to say it everywhere for every type.

Reply:
Hey! I think I made the change for both `v1alpha2` and `v1beta1`... Unless I pushed it wrong